puppet/modules/slave/manifests/rvm.pp

```diff
@@ -17,32 +17,52 @@
   Class['Rvm::System']
 
   if $facts['rvm_installed'] {
+    if $facts['os']['release']['major'] == '9' {
+      exec { 'install openssl 1.1 through rvm':
+        command  => 'rvm pkg install openssl',
+        creates  => '/usr/local/rvm/usr/lib/libssl.so',
+        path     => '/usr/local/rvm/bin:/usr/bin:/bin',
+        provider => 'shell',
+        before   => Slave::Rvm_config['ruby-2.7', 'ruby-3.0'],
+      }
+
+      $build_opts = '--with-openssl-dir=/usr/local/rvm/usr'
+    } else {
+      $build_opts = undef
+    }
+
     rvm::system_user { 'jenkins':
       create  => false,
       require => User['jenkins'],
     }
 
     slave::rvm_config { 'ruby-2.4':
-      version => 'ruby-2.4.3',
+      version    => 'ruby-2.4.3',
+      build_opts => $build_opts,
     }
     slave::rvm_config { 'ruby-2.5':
-      version => 'ruby-2.5.1',
+      version    => 'ruby-2.5.1',
+      build_opts => $build_opts,
```

Comment:
I'd rather remove these instead of trying to build them: https://github.com/theforeman/foreman-infra/pull/2018

Reply:
https://github.com/theforeman/foreman-infra/pull/2019 cleans those up too